src/app/api/quote/route.ts

```diff
@@ -0,0 +1,141 @@
+import {
```

Comment:
just a general thought, but if the goal here is to serve the frontend only with some convenient data, then we could consider trpc

Reply:
Goal is to have a (public) endpoint that can be consumed from any type of client (whatever the external partners prefer). Mostly it will be other backends, I assume.